api_branch_restrictions.go

```diff
@@ -566,127 +567,42 @@ Creates a new branch restriction rule for a repository.  &#x60;kind&#x60; descri
 
 @return Branchrestriction
 */
-func (a *BranchRestrictionsApiService) RepositoriesWorkspaceRepoSlugBranchRestrictionsPost(ctx context.Context, body Branchrestriction, repoSlug string, workspace string) (Branchrestriction, *http.Response, error) {
-	var (
```

Comment:
A lot of these default declaration were not actually used. As such, any unused (only initialized with a default value) variable is declared inline in the `client.prepareRequest()` call.